src/azure/cli/_config.py

```diff
@@ -0,0 +1,55 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+import os
+from six.moves import configparser
+
+GLOBAL_CONFIG_PATH = os.path.expanduser(os.path.join('~', '.azure', 'az_config'))
+ACTIVE_ENV_CONFIG_PATH = os.path.expanduser(os.path.join('~', '.azure', 'env_config', 'default'))
+ENV_VAR_PREFIX = 'AZURE_CLI_'
```

Comment:
The existing environment variables for storage do not use an AZURE_CLI prefix. They use AZURE_STORAGE_ and follow the names used by Xplat for cross-compatibility. If we can justify deviating from Xplat here then I'm fine with it--otherwise just prefix with AZURE_.